src/components/homesections/homesections.css

```diff
@@ -22,3 +22,10 @@
     text-overflow: ellipsis;
     overflow: hidden;
 }
+
+.noLibraryContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
```

Comment:
I added a generic class called `centerMessage` that could be used instead of this one.